README.md

```diff
@@ -46,18 +40,27 @@ Please contact the developers before using it: [simtools-developer@desy.de](mail
 
 ## Quick start
 
-:::{caution}
-The pip-installation of simtools provides limited functionality only
-and is not as well tested as the conda/mamba installation.
-:::
+### Installation for users
 
-Install simtools with pypi (recommended for users):
+Install simtools with pypi:
 
 ```bash
 pip install gammasimtools
 ```
 
-Install simtools with mamba (recommended for developers):
+or using conda/mamba:
+
+```bash
+conda env create -n gammasimtools
+conda install gammasimtools --channel conda-forge
+conda activate gammasimtools
+```
+
+See the [user's guide](https://gammasim.github.io/simtools/user-guide/getting_started.html) for more details.
+
+### Installation for developers
+
+Install simtools from GitHub with conda/mamba:
 
 ```bash
```

Comment:
This is a duplication from the developer guidelines. The link should be enough here.

Reply:
Ok if you prefer this way we can let it. Just remembering to update both in case.